src/public/style/style.css

```diff
@@ -300,4 +301,68 @@ message .-message p {
 /* Some utils */
 .avatar {
   border-radius: 10px;
+}
+
+
+/* Style code blocks */
+pre {
+    font-family: "Courier 10 Pitch", Courier, monospace;
+    font-size: 95%;
+    line-height: 140%;
+    white-space: pre;
+    white-space: pre-wrap;
+    white-space: -moz-pre-wrap;
+    white-space: -o-pre-wrap;
+}       
+
+code {
+    font-family: Monaco, Consolas, "Andale Mono", "DejaVu Sans Mono", monospace;
+    font-size: 95%;
+    line-height: 140%;
+    white-space: pre;
+    white-space: pre-wrap;
+    white-space: -moz-pre-wrap;
+    white-space: -o-pre-wrap;
+    background: #faf8f0;
+
+    display: inline;
+    padding: 0.25em 0.5em;
+
+    letter-spacing: -1.159999996423721px;
+    -moz-tab-size: 4;
+    -o-tab-size: 4;
+    tab-size: 4;
+    text-align: left;
+    text-rendering: optimizeLegibility;
+    -webkit-box-shadow: inset 0 0 0 .1em #eff0f2;
+    box-shadow: inset 0 0 0 .1em #eff0f2;
+    border-radius: 4.80000019073486px 4.80000019073486px 4.80000019073486px 4.80000019073486px;
+    font: normal normal 200 normal 16px/28px Consolas, 'Andale Mono WT', 'Andale Mono', 'Lucida Console', 'Lucida Sans Typewriter', 'DejaVu Sans Mono', 'Bitstream Vera Sans Mono', 'Liberation Mono', 'Nimbus Mono L', Monaco, 'Courier New', Courier, monospace;
```

Comment:
Keen eyes dude, i got them from Medium, and I think that if medium do it, it could not be bad to follow them :) 

By the way, I reduced the numbers of font declarations!